### ISSUE TITLE
[Docs] Lengthscale for each task in LCMvariationalstrategy. 

### ISSUE THREAD

**Issue by srn284** [cornellius-gp/gpytorch#1420]
I have developed a neural network + LMC var strategy model for multitask prediction. The model has 2 tasks and i pass in the inducing points (also shown below). The ard dim is set as 1 right now. 

I'm a bit lost on how the lengthscales function for this multitask LMC modeling. I was wondering how to make every task have it's own lengthscale parameter? I can edit ard_num_dims for the covariance definition below but this cannot be configured to num_Tasks. I see in the code base there is a lmc_coefficient term but could not understand if this does what i'm asking for or something else? I'm overall a little lost and would love some clarity/documentation on this matter. Thank you very much. 

```
class MultitaskGPModel_LCM(gpytorch.models.ApproximateGP):
    def __init__(self, inducing_points, config= None):
        # currently inducing points (latent_dim, numInducingPt, inputSize)

        num_tasks =2
        num_latent = inducing_points.shape[0] 


        print(inducing_points.shape)
        # We have to mark the CholeskyVariationalDistribution as batch
        # so that we learn a variational distribution for each task
        variational_distribution = gpytorch.variational.CholeskyVariationalDistribution(
            inducing_points.size(-2), batch_shape=torch.Size([num_latent])

        )

        # We have to wrap the VariationalStrategy in a MultitaskVariationalStrategy
        # so that the output will be a MultitaskMultivariateNormal rather than a batch output
        variational_strategy = gpytorch.variational.LMCVariationalStrategy(
            gpytorch.variational.VariationalStrategy(
                self, inducing_points, variational_distribution, learn_inducing_locations=True
            ),
            num_tasks=2,
            num_latents=num_latent,
            latent_dim=-1
        )
      

        super().__init__(variational_strategy)

        # The mean and covariance modules should be marked as batch
        # so we learn a different set of hyperparameters
        self.mean_module = gpytorch.means.ConstantMean(batch_shape=torch.Size([num_latent]))
        self.covar_module = gpytorch.kernels.ScaleKernel(
            gpytorch.kernels.MaternKernel(batch_shape=torch.Size([num_latent]),
            batch_shape=torch.Size([num_latent])
        )

    def forward(self, x):
        # The forward function should be written as if we were dealing with each output
        # dimension in batch
        mean_x = self.mean_module(x)

        covar_x = self.covar_module(x)
        sout = gpytorch.distributions.MultivariateNormal(mean_x, covar_x)
        return sout

```

Gpytorch: 1.2.1
pytorch: 1.6.0
